docs/content/tutorials/tutorial-add-radius/index.md

```diff
@@ -0,0 +1,303 @@
+   pod/redis-replica-787cd488b4-vf6fh   1/1     Running   0          10s
+   pod/frontend-77dfc58d7c-8zht6        1/1     Running   0          10s
+   pod/redis-master-7597b47b98-ddgq9    1/1     Running   0          10s
+   pod/frontend-77dfc58d7c-2gs2p        1/1     Running   0          10s
+   pod/frontend-77dfc58d7c-2qd22        1/1     Running   0          10s
+
+   NAME                    TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)        AGE
+   service/frontend        NodePort    10.43.49.106    <none>        80:30972/TCP   10s
+   service/redis-master    ClusterIP   10.43.233.148   <none>        6379/TCP       10s
+   service/redis-replica   ClusterIP   10.43.118.183   <none>        6379/TCP       10s
+
+   NAME                            READY   UP-TO-DATE   AVAILABLE   AGE
+   deployment.apps/redis-replica   2/2     2            2           10s
+   deployment.apps/redis-master    1/1     1            1           10s
+   deployment.apps/frontend        3/3     3            3           10s
+
+   NAME                                       DESIRED   CURRENT   READY   AGE
+   replicaset.apps/redis-replica-787cd488b4   2         2         2       10s
+   replicaset.apps/redis-master-7597b47b98    1         1         1       10s
+   replicaset.apps/frontend-77dfc58d7c        3         3         3       10s
+   ```
+
+4. Validate that the application is running as expected:
+
+   Run this command to port forward the Guestbook application to port `8080` on your local machine:
+
+   ```bash
+   kubectl port-forward -n demo svc/frontend 8080:80
+   ```
+
+   Open a browser and navigate to [`http://localhost:8080`](http://localhost:8080). You should see the Guestbook application running:
+
+   <img src="./guestbook-ui.png" alt="Guestbook application" width=500>
+
+   Terminate the port forwarding process by pressing `CTRL+C` in your terminal.
+
+5. Confirm that Radius has not yet been added:
+
+   Run this command to view the state of your application using Radius:
+
+   ```bash
+   rad app connections -a demo
+   ```
+
+   Since you have not yet added Radius to the application, the `rad` CLI will not recognize your `demo` application and you should see a message in the output similar to:
+
+   ```
+   Application "demo" does not exist or has been deleted.
+   ```
+
+## Step 3: Add Radius to the Guestbook application
+
+You will now add Radius to the Guestbook application's Kubernetes deployment manifests by making edits to the YAML files in the `deploy` directory. 
+
+1. In each of the YAML files, add the `annotations` property to `metadata`, and then add the `rad.app/enabled: 'true'` annotation. Note that the `'true'` must be surrounded in quotes.
+
+   ```yaml
+   ...
+   metadata:
+      ...
+      annotations:
+         radapp.io/enabled: 'true'
+      ...
+   spec:
+   ...
+   ```
+
+   You should add the annotation to the following files:
+      - `deploy/frontend-deployment.yaml`
+      - `deploy/frontend-service.yaml`
```

Comment:
This doesn't do anything for the services.

Reply:
Good point, then it probably makes sense to exclude `Services` from the set of items to modify. I will make the change.